Core/Exercises/Models/ExerciseDto.cs

```diff
@@ -6,6 +6,7 @@ public record ExerciseDto(
     string Name,
     string Description,
     string Solution,
+    int Language,
```

Comment:
Suggest a naming referring to the Solution, as this could be understood as the language the exercise must be solved in

`SolutionLanguage` or something similar